doc/manual/castxml.1.rst

```diff
@@ -99,3 +99,52 @@ case, CastXML always adds the following predefined macros:
 
 Source files may use these to identify the tool that is actually doing the
 preprocessing even when ``--castxml-cc-<id>`` changes the predefined macros.
+
+FAQ
+===
+
+Why are C++ function bodies not dumped in XML?
+----------------------------------------------
+
+The original sponsors of the project had no need for function bodies.
+So far the authors have not had time or funding to implement the support.
+
+Is there a DTD specifying the XML format dumped?
+------------------------------------------------
+
+No.
+
+Why are most classes, structs, and enums marked as "artificial"?
+----------------------------------------------------------------
+
+The artificial attribute marks nodes that are artificially generated by the compiler
+(such as implicit copy constructors). The C++ code
+
+ | struct foo {};
+
```

Comment:
The code block syntax should be:

```rst
.. code-block:: c++

  struct foo {};
```

Please revise this and the other instances.